.github/workflows/automerge.yml

```
@@ -0,0 +1,37 @@
+name: Automerge
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+env:
+  # replace "github_username" with your GitHub username
+  # replace "github.com/username/repo.git" with your GitHub repo path
+  # do NOT replace ${{secrets.GITHUB_TOKEN}}, GitHub will take care of it
+  MY_REPO: https://GeospatialResearch:${{secrets.GITHUB_TOKEN}}@github.com/GeospatialResearch/Digital-Twins-191.git
+
+  # replace "long-lived_branch_name" with your branch name
+  LONG_LIVED_BRANCH: master
+
+  # replace it with the path to master repo
+  MASTER_REPO: https://github.com/GeospatialResearch/Digital-Twins-191.git
```

```suggestion
  MY_REPO: https://GeospatialResearch:${{secrets.GITHUB_TOKEN}}@github.com/GeospatialResearch/Digital-Twins.git

  # replace "long-lived_branch_name" with your branch name
  LONG_LIVED_BRANCH: master

  # replace it with the path to master repo
  MASTER_REPO: https://github.com/GeospatialResearch/Digital-Twins.git
```
In the real repo we need to update these values sothey don't use the fork